projects/Mallard/src/theme/typography.ts

```diff
@@ -161,6 +161,12 @@ const scale = {
                 lineHeight: 50,
             },
         },
+        2.25: {
```

Comment:
floats as keys make me a little 😬  
might be worth explicitly making these string keys?